databuilder/codes.py

```diff
@@ -34,30 +39,84 @@ def _to_primitive_type(self):
 class BNFCode(BaseCode, system_id="bnf"):
     "Pseudo BNF"
 
+    regex = re.compile(
+        r"""
+        # Standard BNF code
+          # Chapter, Section, Paragraph, Sub-paragraph
+          [01][0-9]{6}
+          # Chemical
+          [0-9A-Z]{2}
+          # Product, strength-formulation, generic equivalent
+          ([A-Z][0-9A-Z]){3}
+        | # OR
+        # Appliances
+        2[0-3][0-9]{9}
+        """,
+        re.VERBOSE,
+    )
+
 
 class CTV3Code(BaseCode, system_id="ctv3"):
     "CTV3 (Read V3)"
 
-
-class DMDCode(BaseCode, system_id="dmd"):
-    "Dictionary of Medicines and Devices"
+    # Some of the CTV3 codes in OpenCodelists violate the below, either by having a
```

Comment:
Clarify that this is in the OpenCodelists coding system data, and not in any codelists?

Reply:
Thanks, good suggestion.